src/transfer/messages.rs

```diff
@@ -36,7 +36,7 @@ pub struct CancelRequest {
 #[derive(Serialize, Deserialize, Debug)]
 pub enum JobSelector {
     All,
-    LastN(usize),
+    LastN(JobId),
```

Comment:
It feels weird that this is `JobId` :)

Reply:
Ok, we can alias is as JobCounts, but technically it should be the same type.

Reply:
I don't think that it should be the same type necessarily, you just want N last jobs, which corresponds to `usize`. The fact that you implement this interface with subtraction of `JobId`s shouldn't affect the interface :)

Reply:
I strongly disagree. usize has platform dependent-size, and you definitely do not want to have a protocol which shape depends on a platform. "Please send 32b if your server is running on platform X or 64b if on platform Y, btw: you do not know what platform is you peer" is worse then Dask protocol:)

Reply:
Oh, ok, I mainly meant "an unsigned integer", `usize` is indeed impractical. So, let's say `u64`. Still, there should be no relation to `JobId` :)